admin/webUI/app/components/invite/inviteController.js

```diff
@@ -200,6 +200,8 @@
 	}
 	
 	function candidatesController($rootScope, $stateParams, $state, inviteService) {
+		$rootScope.sortType = 'score';
+		$rootScope.sortReverse = true;
 		candidatesVm = this;
```

Comment:
How do I access it from within the template, as candidateVm.sortType or only sortType would also work?